src/System.Net.Requests/tests/RequestStreamTest.cs

```diff
@@ -11,11 +11,18 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Tests hang on .NET Framework")]
     public class RequestStreamTest
     {
         readonly byte[] buffer = new byte[1];
 
+        static RequestStreamTest()
+        {
+            // On .NET Framework, the default limit for connections/server is very low (2).
+            // On .NET Core, the default limit is higher. Since these tests run in parallel,
+            // the limit needs to be increased to avoid timeouts when running the tests.
+            ServicePointManager.DefaultConnectionLimit = int.MaxValue;
```

Comment:
This is the bug in desktop we had the discussion about like a month ago when we opened #16928. This is not a leak, it is because in Desktop we open a connection to the server but in Core buffer the content as if in desktop .AllowWriteStreamBuffering was set to true. (https://github.com/dotnet/corefx/issues/11873#issuecomment-248124938). So maybe another workaround will be setting .AllowWriteStreamBuffering to true and disposing the WebRequest so that it is actually written to the server?